### DISCUSSION TITLE
Bold font in math mode

### DISCUSSION THREAD

**Discussion by hasky67** [olsak/OpTeX#53] [Q&A]
Hi,
I am strugling with writting bold font in math mode. Please, do I miss something? In text mode it works fine.
`$$	\eqalign{\dot{p} &= {\bf v} \cr
			 \dot{v} &= R(a_m-a_b) + g} \eqmark $$`

But during compilation, it gives me
`! Undefined control sequence.
<argument> \_bsansvariables \_sansgreek 
                             \_bsansdigits 
\_inmath #1->\_relax \_ifmmode #1
                                 \_fi 
<argument> \dot {p} &= {\bf 
                 v} \cr \dot {v} &= R(a_m-a_b) + g
\eqalign ...}$\_hskip .5\_eqspace \_hfil \_crcr #1
                                                  \_crcr }}\_thinsk 
l.44 			 \dot{v} &= R(a_m-a_b) + g}
                                  \eqmark $$
? 
`
Any help please?

Thanks

**Comment by olsak** [comment]
The problem is not reproducible. Your line
```
$$ \eqalign{\dot{p} &= {\bf v} \cr \dot{v} &= R(a_m-a_b) + g} \eqmark $$
```
is OK. It does not generate any error. Give full example.
The error message is not fully copied. The most important thing: where the lines in the error message end, is missing. So, we cannot guess what control sequence is undefined.

**Comment by olsak** [comment]
The `\_sansgreek` was renamed to `\_bsansgreek` in commit https://github.com/olsak/OpTeX/commit/b7dc0e94ceb541565ca7bfd1d36b3cef0e2cd93d#diff-4c7648fdf8b4da4cca53d5103a0f13490bfcf5f1b50d78a23a9c2af48f4ec9f5 Maybe, you are using macros which were valid before this commit.
